packages/terra-form-select/src/_Menu.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import { polyfill } from 'react-lifecycles-compat';
 import 'terra-base/lib/baseStyles';
-import { KeyCodes, Variants } from './_constants';
+import { KeyCode, Variants } from './_constants';
```

Comment:
I feel like we should directly import from the keycode package and remove the keycode exports from `_constants.js`